packages/dashboard-hardhat-plugin/README.md

```diff
@@ -1,62 +1,124 @@
 # @truffle/dashboard-hardhat-plugin
 
-Enable decoded results when using [Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) with your Hardhat projects.
+> No more private keys inside `.env` files! Truffle Dashboard lets you sign
+> messages with MetaMask and provides insights for better dapp development.
+> :lock:
 
-## What
+This plugin makes it easy to access all the features of
+[Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/),
+including better security and high-quality transaction decoding.
 
-This plugin enables you to see decoded transaction information (both the function signature and the values of any arguments passed) when using [Truffle Dashboard](https://trufflesuite.com/docs/truffle/how-to/use-the-truffle-dashboard/) with your Hardhat projects.
+## Summary
 
-It extends `npx hardhat compile` by sending the compiled artifacts to Truffle Dashboard, which in turn uses [@truffle/decoder](https://trufflesuite.com/docs/truffle/codec/modules/_truffle_decoder.html) for decoding, giving you that extra degree of visibility before signing the transactions with your browser-based wallet.
+This plugin automatically adds a `--network truffleDashboard` to your Hardhat
+project and tells Truffle Dashboard all the information it needs about your
+smart contracts.
+
+This plugin enables you to see decoded transaction information (both the
```

Comment:
Extremely minor nitpick with the dupicate opening to the paragraph. Wonder if "It also enables you..." might read a little better?

Reply:
I'll try that. I know what you mean though.

Reply:
I did another pass over the whole README. Hopefully this is better now!